ipfx/epochs.py

```diff
@@ -30,27 +41,16 @@ def get_last_vm_epoch(idx1, hz):
     return idx0, idx1
 
 
-def get_first_vm_noise_epoch(idx, hz):
+def get_first_noise_epoch(idx, hz):
 
     return idx, idx + int(NOISE_EPOCH * hz)
```

Comment:
Again another double check `int()` will always truncate towards zero, is that the 'right' behavior?